### DISCUSSION TITLE
FDC100VNP support

### DISCUSSION THREAD

**Discussion by matheuscanela** [SwiCago/HeatPump#219] [General]
Hey folks!

I am trying to figure out if my unit FDC100VNP supports the CN105. I don't have access to the unit yet, so I want to know if anyone has a similar unit.

I contacted Mitsubishi, and they said that I need the unit below (MH-RC-WIFI-1B). However, I checked the installation manual of the WIFI unit, but it only requires two cables instead of the CN105, so I'm not sure about it.

Mitsubishi hardware is not expensive, but having an Arduino reading and changing the machine would be better :)

[MH-RC-WIFI-1B_Install Manual.pdf](https://github.com/SwiCago/HeatPump/files/15399135/MH-RC-WIFI-1B_Install.Manual.pdf)

**Comment by fastbike** [comment]
Same name but a different company.
Your unit is Mitsubishi Heavy Industries, not Mitsubishi Electric.
I think there might a different project if you google it.

**Reply by matheuscanela** [reply]
No worries, thanks!

I found it 
https://github.com/absalom-muc/MHI-AC-Ctrl
